src/styles/components/navbar.scss

```diff
@@ -79,31 +53,9 @@
       transition-property: transform;
     }
 
-    .single-link{
+    & > a{
```

Comment:
```suggestion
    > a{
```
or even better:
```suggestion
    .btn {
```

Also the size seems a bit off

![Screenshot 2023-05-08 at 21 11 04](https://user-images.githubusercontent.com/2543423/236913061-df1f9fed-e286-4bd0-93e0-e5104d231655.png)